.github/workflows/release.yml

```diff
@@ -3,10 +3,30 @@ name: Release
 on:
   workflow_dispatch:
     inputs:
+      releaseType:
+        description: "Type of release"
+        required: true
+        type: choice
+        options:
+          - patch
+          - minor
```

Comment:
We agreed we're not going to support automation for major releases at the moment. We'll do them manually.
We can revisit this if needed though.

Reply:
Ok, I think we can "hack" the workflow during the 1.0.0 release branch so that it worked as worked before - stopped in the middle so we could change the catalog manually.